.github/ISSUE_TEMPLATE/releases/release_checklist.md

```diff
@@ -49,37 +49,34 @@
           - NuGet-client (except for feature band updates)
           - fsharp (except for feature band updates)
           - VSTest (except for feature band updates)
-1.  - [ ] [2.1] Fetch internal git data for submodules.
-      1.  Run `git submodule update --init --recursive` and see errors due to missing commits.
-      1.  Run `fetch-vsts-commits.sh` with a PAT to automatically fetch internal refs.
-      1.  Run `git submodule update --init --recursive` again to update failed submodules.
-          * If some refs are still missing on unusual repos, those repos may need to be set up in `fetch-vsts-commits.sh`.  Note: Repos in `fetch-vsts-commits.sh` don't need to be cleaned up.  
 1.  - [ ] Regenerate patches as necessary.
```

Comment:
This should call out in more detail what it means to regenerate patches.

Reply:
Added a new doc with with details.